modules/util/source-formatter/src/main/java/com/liferay/source/formatter/check/UpgradeCommerceOrderConstantsCheck.java

```diff
@@ -0,0 +1,50 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2023 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.source.formatter.check;
+
+import com.liferay.petra.string.StringUtil;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+/**
+ * @author Lázaro Costa
+ */
+public class UpgradeCommerceOrderConstantsCheck
+	extends BaseUpgradeMatcherReplacementCheck {
+
+	@Override
+	protected String formatMatcherIteration(
+		String content, String newContent, Matcher matcher) {
+
+		String constant = matcher.group(1);
+		String constantCall = matcher.group();
+		String newConstantCall = "CommerceOrderPaymentConstants.";
```

Comment:
@lazhcsilva, the SF doesn't complain about this? Did you run? Just to make sure.